### ISSUE TITLE
Kubernetes container "kube-proxy" not available

### ISSUE THREAD

**Issue by Cryptophobia** [mantl/mantl#1869]
Hello! have been looking deeper into the issues with starting the kubernetes-dashboard and the kube-dns-v8-**** pods inside vagrant. When starting mantl with vagrant this is the state of kubernetes pods after the `vagrant up` command:

```
[vagrant@control-01 manifests]$ kubectl get pods
NAME                                 READY     STATUS              RESTARTS   AGE
kube-apiserver-control-01            1/1       Running             1          1h
kube-controller-manager-control-01   1/1       Running             4          1h
kube-dns-v8-rfuth                    0/2       ContainerCreating   0          1h
kube-podmaster-control-01            2/2       Running             2          1h
kube-proxy-control-01                1/1       Running             1          1h
kube-proxy-kubeworker-001            0/1       Pending             0          1h
kube-scheduler-control-01            1/1       Running             5          1h
kube2consul-control-01               1/1       Running             3          1h
kubernetes-dashboard-gk8lx           0/1       ContainerCreating   5          1h
```

I have traced it down to the following problem when running the command: 

`kubectl logs kube-proxy-kubeworker-001`

Output:
`Error from server: container "kube-proxy" in pod "kube-proxy-kubeworker-001" is not available` 

@stevendborrelli Is the kubernetes image `"ciscocloud/hyperkube-amd64:v1.3.3-consul"` not available anymore or something❓

**Comment by stevendborrelli** [comment]
The image is available, I just pulled it down. 

We may want to look at moving this repo into mantl and updating it to be a newer version: https://github.com/asteris-llc/kubernetes-packaging

**Comment by Theaxiom** [comment]
This is related to #1873

**Comment by thomasvincent** [comment]
this is a quick fix when we get access to the mantl docker repo

**Comment by Theaxiom** [comment]
- https://github.com/mantl/kubernetes-packaging

**Comment by Theaxiom** [comment]
This is fixed in master with #1963
